### PR TITLE
improve failure message sent by check_chronos_jobs

### PR DESCRIPTION
**Pull request by Rob-Johnson**
add instructions for rerunning the job, viewing logs and running
paasta status

### CODE REVIEW

**Comment by Rob-Johnson** [comment]
screen shot 2016-04-28 at 10 05 07

**Comment by Rob-Johnson** [comment]
probably worth adding some \n

File: paasta_tools/check_chronos_jobs.py [hunk 1]
```diff
@@ -107,18 +107,31 @@ def build_service_job_mapping(client, configured_jobs):
...
 
 
-def message_for_status(status, service, instance, full_job_id):
+def message_for_status(status, service, instance, cluster):
     if status not in (pysensu_yelp.Status.CRITICAL, pysensu_yelp.Status.OK, pysensu_yelp.Status.UNKNOWN):
         raise ValueError('unknown sensu status: %s' % status)
     if status == pysensu_yelp.Status.CRITICAL:
-        return 'Last run of job %s%s%s Failed - job id %s' % (service, utils.SPACER, instance, full_job_id)
+        return (
+            "Last run of job %(service)s%(separator)s%(instance)s failed. "
```
Comment (solarkennedy) [inline paasta_tools/check_chronos_jobs.py]: Can you add newlines so it doesn't look like a massive block of text?

File: paasta_tools/check_chronos_jobs.py [hunk 2]
```diff
@@ -107,18 +107,31 @@ def build_service_job_mapping(client, configured_jobs):
...
+            "You can view the logs for the job with paasta logs -s %(service)s -i %(instance)s -c %(cluster)s ."
+            "If your job didn't manage to start up, you can view the stdout and stderr "
+            "of your job using paasta status -s %(service)s -i %(instance)s -c %(cluster)s -vv ."
+            "If you need to rerun your job for the datetime it was started, you can do so with "
+            "paasta rerun -s %(service)s -i %(instance)s -c %(cluster)s -d {datetime} ."
+            "See the docs on paasta rerun for more details."
```
Comment (solarkennedy) [inline paasta_tools/check_chronos_jobs.py]: you can link to.. something.. not sure what though

**Comment by solarkennedy** [comment]
ship with lots of newlines please

**Comment by Rob-Johnson** [comment]
screen shot 2016-04-29 at 09 34 28

**Pull request merged**